docs/user-guide/plugins/github-pull-request-updater.md

```diff
@@ -0,0 +1,58 @@
+# GitHub Pull Request Updater
+
+## Introduction
+The **GitHub Pull Request Updater** plugin automatically adds pipeline status and deployment information as comments on your GitHub pull requests. Every time your Devtron CI pipeline runs, you get instant visibility onto the specific pull request on GitHub. This seamless integration eliminates the need to switch between multiple tools or platforms to track your deployment, Development teams also saves their time by avoiding manual status updates and reducing the back-and-forth communication about deployment states. 
+
+### Prerequisites
+Before integrating the **GitHub Pull Request Updater** plugin, ensure you have an GitHub Personal Access Token (PAT) with appropriate permissions to access the your GitHub repository.
+
+---
+
+## Steps
+1. Go to **Applications** → **Devtron Apps**.
+2. Click your application.
+3. Go to **App Configuration** → **Workflow Editor**.
+4. Click **New Workflow** and navigate to the **Build and Deploy from Source Code**.
+5. Fill the required fields in the **Create build pipeline** window and navigate to the **Post-build stage**.
+
+{% hint style="warning" %}
+If you have already configured workflow, edit the build pipeline, and navigate to **Post-build stage**.
+{% endhint %}
+
+6. Under 'TASKS', click the **+ Add task** button.
+7. Click the **GitHub Pull Request Updater** plugin.
+8. Enter the following [user inputs](#user-inputs) with appropriate values.
+---
+
+## User Inputs
+
+### Task Name
+Enter the name of your task
+
+e.g., `GitHub Pull Request Updater`
+
+### Description
+Add a brief explanation of the task and the reason for choosing the plugin. Include information for someone else to understand the purpose of the task.
+
+e.g., `A plugin to update GitHub pull request`
+
+### Input Variables
+
+| Variable                 | Format       | Description | Sample Value |
+| ------------------------ | ------------ | ----------- | ------------ |
+|   AccessToken             | STRING      | GitHub Personal Access Token (PAT) with appropriate permissions, used for authentication.  | ghp_1234abcd5678efgh9012ijkl3456mnop            |
+|   UpdateWithDockerImageI  | BOOL       | When set to true, the plugin will include Docker image information in PR comments. Default: True        |  true |
```

Comment:
It should be `UpdateWithDockerImageId` instead of `UpdateWithDockerImageI`

Reply:
done